lang/fpc/Portfile

```diff
@@ -263,7 +263,7 @@ if {${subport} eq "${name}"} {
 
     if       {${os.major} >= 11}   { # Big Sur
```

Comment:
This instance of `os.major` isn't right either, and neither is the one later in the file. Big Sur isn't Darwin 11; it's Darwin 20.

As I explained in #19837, it is customary to use `os.major`, not `os.version` or `macos_version`.

Also, verify that `${os.platform} eq "darwin"` before comparing `os.major`.

Reply:
@ryandesign In its existing form the port should be set to `macosx` perhaps, and no need to check for `darwin`, since it does not support BSD and Linux, which need separate bootstrap compilers.

Reply:
As pointed out by barracuda156, the bootstrap compiler makes the package is actually macOS only. Correct me, if I am wrong, but I should change platform from darwin to macosx. 

Regarding version numbers:
I find the use of Darwin version very clumsy. It is hard enough to keep track of the relation between name and version number of the macOS version and the Darwin version adds another layer of complexity on top of that. You say,  it is customary to use. I see no benefit to use this custom here, rather the contrary. According to the MacPorts Guide using macos_version_major does exactly, what I want. 
Correct me if I understand this wrong, but the direct comparison should work with versions > 11 and the use of vercmp is required for 10.x only.

Reply:
@kamischi 
10.5 = 9
10.6 = 10
10.7 = 11
etc.
There is no need for vercomp, just use `os.major`.

Reply:
I am lazy and bad in memorizing. So, I would have to add that list as a note to the package description, or google it whenever I need it, or yada, yada ... Maybe this makes you understand, what I mean by clumsy 😉 
No offence intended.